ansible/roles/ocp4-workload-acm-app-policies-lab/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+become_override: False
+ocp_username: opentlc-mgr
+silent: False
+
+_acm_project: "open-cluster-management"
```

Comment:
Can we rename these variables to use the workload name as the prefix? And rename the workload to use underscores?

See roles_workload/ocp4_workload_example for how to.